census/src/main/java/io/grpc/census/CensusTracingModule.java

```diff
@@ -231,6 +226,21 @@ private void recordMessageEvent(
     span.addMessageEvent(eventBuilder.build());
   }
 
+  private void recordAnnotation(
+      Span span, MessageEvent.Type type, int seqNo, boolean isCompressed, long size) {
+    StringBuilder messageTypeBuilder = new StringBuilder("compressed");
```

Comment:
This is a surprising usage of StringBuilder. This seems likely to be left over from earlier experimentation, as I don't see its purpose now. I'd have just expected `String messageType = isCompressed ? "compressed" : "uncompressed"`, which seems clearer and easier to write. Or do the same assignment and prefixing, just using String directly.